lib/github.mjs

```diff
@@ -20,3 +22,37 @@ export const isGitHubContributor = async username => {
 
   return total_count > 0;
 };
+
+export const inviteIfNecessary = async username => {
+  logger.verbose('checking if @%s was already invited to this team', username);
+
+  // Make sure we can make authenticated requests to the GitHub API.
+  github.authenticate({ type: 'token', token: GITHUB_TOKEN });
+  const options = { team_id: GITHUB_TEAM_ID, username };
+
+  try {
+    const response = await github.orgs.getTeamMembership(options);
+
+    if (response.status === 200) {
+      logger.verbose('@%s has already been invited to this team', username);
+      return;
+    }
+  } catch (err) {
+    // If the user hasn’t been invited, Octokit throws with a 404.
+    if (err.code === 404) {
+      const invite = await github.orgs.addTeamMembership(options);
+
+      if (invite.data.state === 'active') {
+        logger.verbose('@%s is already a member of this team', username);
+      } else {
+        logger.verbose('@%s was invited to join the team', username);
+      }
+
+      return;
+    }
+
+    // If we get here, something _actually_ went wrong.
+    logger.error('There was an error inviting @%s to the team', username);
+    logger.error(err);
+  }
+};
```

Comment:
This is code that's apparently been deployed for a while, but was never committed. I am ashamed. 🙈